pack/server/image/Dockerfile

```diff
@@ -152,6 +152,22 @@ COPY --from=sealio/casdoor:v1.344.0-seal.1 \
     /casdoor \
     /usr/bin/
 
+# get helm
+ENV HELM_VERSION="v3.12.3"
+ENV HELM_URL=https://get.helm.sh/helm-${HELM_VERSION}-${TARGETOS}-${TARGETARCH}.tar.gz
+RUN curl ${HELM_URL} | tar xvzf - --strip-components=1 -C /usr/bin
+
+# get prometheus helm chart
+ENV CHARTS_DIR="/var/lib/walrus/manifests/charts"
+RUN set -eux; \
+    mkdir -p ${CHARTS_DIR}
+ENV PROMETHEUS_REPO_URL="https://prometheus-community.github.io/helm-charts"
+ENV PROMETHEUS_CHART_NAME="prometheus"
+ENV PROMETHEUS_CHART_VERSION="23.4.0"
+RUN helm repo add prometheus-community ${PROMETHEUS_REPO_URL} && \
+    helm repo update && \
+    helm pull prometheus-community/${PROMETHEUS_CHART_NAME} --version ${PROMETHEUS_CHART_VERSION} -d ${CHARTS_DIR}
+
```

Comment:
looks like we only need to download a Prometheus chart ball from upstream, can we use [`ADD`]( https://docs.docker.com/engine/reference/builder/#verifying-a-remote-file-checksum-add---checksumchecksum-http-src-dest) to download [the prometheus ball](https://github.com/prometheus-community/helm-charts/releases/download/prometheus-23.4.0/prometheus-23.4.0.tgz)? 

- https://github.com/prometheus-community/helm-charts/releases/tag/prometheus-24.0.0. FYI, 24.0.0 is aboard.